charts/kserve-resources/values.yaml

```diff
@@ -1,5 +1,5 @@
 kserve:
-  version: &defaultVersion v0.11.1
+  version: &defaultVersion v0.12.0-rc0
   modelmeshVersion: &defaultModelMeshVersion v0.11.0
```

Comment:
Do we want to bump this `modelmeshVersion` as well? Seems like a newer version is available: https://github.com/kserve/modelmesh/releases/tag/v0.11.1

Reply:
yes

Reply:
Bumped to v0.11.1